schema-anonymizer/anonymizer.py

```diff
@@ -273,6 +276,8 @@ def transform_if_not_exists(cql_string):
                     help='Replication factor to override original setting. Optional.')
 parser.add_argument('--format', metavar='cql', default='cql', choices=['cql', 'gemini'],
                     help='Output format for the schema. cql or gemini (json). Default is cql.')
+parser.add_argument('--no-anonymize', dest='anonymize', action='store_false', help='Disable anonymization functions')
```

Comment:
This argument was deliberately selected so that anonymizing of schemas is still the default process.  In order to preserve confidentiality of submitted schemas we should anonymize in all cases _except_ when a user has explicitly asked for this not to occur.